app/src/main/res/layout/drag_drop_interaction_items.xml

```diff
@@ -0,0 +1,39 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <data>
+
+    <variable
+      name="htmlContent"
+      type="CharSequence" />
+
+    <variable
+      name="viewModel"
```

Comment:
This doesn't appear to be used--do we need it?

Reply:
yes this is being used for binding/saving the content of html